angelspetrochem/prototypes/items/petrochem-basics.lua

```diff
@@ -86,16 +86,16 @@ data:extend(
       localised_description = show_formula and {"tooltip-description.chemical-formula", "CO"} or nil,
       icons = angelsmods.functions.create_gas_fluid_icon(
         {"__angelspetrochem__/graphics/icons/molecules/carbon-monoxide.png", 72},
-        "ccc"
+        "COC"

```

Comment:
Same comment, leave as CCC? it's less vibrant red (and let it match recipe with fluid icon)